sdk/metricsadvisor/ai-metrics-advisor/swagger/README.md

```diff
@@ -18,7 +18,7 @@ input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/7efe0ae
 add-credentials: false
 override-client-name: GeneratedClient
 use-extension:
-  "@autorest/typescript": "6.0.0-dev.20210223.1"
+  "@autorest/typescript": "6.0.0-beta.4"
```

Comment:
let's just `typescript: true` instead to always use the most recent version of the TS code gen.